src/components/funFacts/FunFactComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { funFacts,  FunFact  as FunFactInterface } from "./funfacts";
+import "./FunFactPage.scss";
+
+interface FunFactProps {
+    funFacts: FunFactInterface[];
+}
+
+export const FunFact: React.FunctionComponent<FunFactProps> = ({ funFacts }) => {
+
+  const [selectedFactId, setSelectedFactId] = useState(0);
+
+  const toggleFactDetail = (funFactId: number) => {
+    if (selectedFactId === funFactId){
+      setSelectedFactId(0);
+    }
+    else {
+      setSelectedFactId(funFactId);
+    }
+  }
+
+  return(
+    <div>
+      {funFacts.map(funFact => (
+        <div className="funfact">
+          <div className="funfact__prompt">
+            <h2 className="funfact__prompt-question">{funFact.prompt}</h2>
```

Comment:
Just realised: "fun fact" is two words!
```suggestion
        <div className="fun-fact">
          <div className="fun-fact__prompt">
            <h2 className="fun-fact__prompt-question">{funFact.prompt}</h2>
```